src/router.js

```diff
@@ -208,19 +217,9 @@ export class Router {
 
     config.navModel = navModel;
 
-    if ((config.nav || 'order' in navModel) && this.navigation.indexOf(navModel) === -1) {
-      navModel.order = navModel.order || config.nav;
-      navModel.href = navModel.href || config.href;
-      navModel.isActive = false;
-      navModel.config = config;
```

Comment:
This was unused, and is no longer copied onto the navModel. The same is true for `settings` above.